backend/main.py

```
@@ -60,6 +68,145 @@ async def root():
+    Validates that all model IDs exist in OpenRouter before saving.
+    """
+    # Validate that we have at least one council model
+    if not request.council_models:
+        raise HTTPException(status_code=400, detail="At least one council model is required")
+    
+    # Validate models exist in OpenRouter
+    try:
+        cache = await get_available_models()
+        
+        # Check council models
+        valid_council, invalid_council = validate_model_ids(request.council_models, cache)
```

Ruff (0.14.14)

[warning] Unpacked variable `valid_council` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)